app/assets/stylesheets/mixins/_colors.scss

```diff
@@ -0,0 +1 @@
+$header_title_color: #fefeff;
```

Comment:
Files should end with a trailing newline<br>Name of variable `header_title_color` should be written in all lowercase letters with hyphens instead of underscores